### PR TITLE
feat: 0.36.1

### PR DESCRIPTION
**Pull request by caspiano**
- Adds support for crystal 0.36.1
- Adds binaries and scripts to the path

### CODE REVIEW

File: shard.override.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+dependencies:
+  retriable:
+    github: Sija/retriable.cr
+  awscr-signer:
```
Comment (xxkb) [inline shard.override.yml:4]: What the reason for the override? Looks like it's a first-party - any reason the different ref can't be listed straight in `shard.yml`?

Comment (caspiano) [reply shard.override.yml:4]: it's a temporary fix, and not an explicit dependency

**Comment by w-le** [comment]
Cool i'll test a deploy using partner-environment and placeos/init:feat/0.36.1 image

**Review by xxkb — APPROVED** [review]
Build looks green. If @w-le testing goes without issue, gtg.

**Pull request merged**